### ISSUE TITLE
default account ?

### ISSUE THREAD

**Issue by listener-He** [long2ice/meilisync-admin#7]

**Comment by isqilinlin** [comment]
+1

**Comment by boolean-dev** [comment]
+1

**Comment by xAiDAx** [comment]
I found a workaround. you can add an admin with this sql statement. the password is a bcrypt  hash

`INSERT INTO admin ( nickname, email, password, is_superuser, is_active) VALUES
("", "admin@nomail.com", "$2a$12$lHPmL6yncUwa/WSb3nvqzeCZtWQ7ORZdkZsbyOWul7Nk0SIB7rnjS", 1, 1);
`

User: admin@nomail.com
password: password

**Comment by Sinitra** [comment]
+1

**Comment by mtian999** [comment]
post 访问接口/api/admin header Content-Type:application/json  body json{
  "email": "you email",
  "password": "you password",
  "nickname": "you name"
}

**Comment by FreddyJohn** [comment]
I agree this needs to be added, it makes a total of 0 sense that the recommended docker defaults to an admin page with auth without a default user, either provide an option to remove the auth for local development or provide a default user. Very annoying
